routes/indicators/create_indicator.js

```diff
@@ -36,6 +36,9 @@ module.exports = [
           return res({id: ret[0]});
         })
         .catch(function (err) {
+          if (err.code === '23505') {
```

Comment:
This error corresponds to unique key violation in the database, I can add a comment https://www.postgresql.org/docs/current/static/errcodes-appendix.html